modules/health_quest/spec/request/apidocs_request_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe 'vaos appointment requests', type: :request do
```

Comment:
Is this a `VAOS` appointment request? I don't understand the naming discrepancy between "health_quest" and this spec

Reply:
I've removed vaos and set it to health quest. This should remove any confusions and or discrepancies.